Application/EdFi.Ods.Security/Authorization/AuthorizationSegmentSqlProviderBase.cs

```diff
@@ -59,28 +59,28 @@ public AuthorizationSegmentSqlProviderBase(IAuthorizationViewsProvider authoriza
                 foreach (var claimEndpointsWithSameName in claimEndpointsGroupedByName)
                 {
                     string claimEndpointName = claimEndpointsWithSameName.Key;
-                    string targetEndpointName = authorizationSegment.TargetEndpoint.Name;
+                    string subjectEndpointName = authorizationSegment.SubjectEndpoint.Name;
 
-                    var targetEndpointWithValue =
-                        authorizationSegment.TargetEndpoint as AuthorizationSegmentEndpointWithValue;
+                    var subjectEndpointWithValue =
+                        authorizationSegment.SubjectEndpoint as AuthorizationSegmentEndpointWithValue;
 
                     // This should never happen
-                    if (targetEndpointWithValue == null)
+                    if (subjectEndpointWithValue == null)
                     {
                         throw new Exception(
-                            "The claims-based authorization segment target endpoint for a single-item authorization was not defined with a value.");
+                            "The claims-based authorization segment subject endpoint for a single-item authorization was not defined with a value.");
                     }
 
-                    if (targetEndpointWithValue.Value == null)
+                    if (subjectEndpointWithValue.Value == null)
                     {
                         throw new EdFiSecurityException(
-                            $"Access to the resource item could not be authorized because the '{targetEndpointWithValue.Name}' of the resource is empty.");
+                            $"Access to the resource item could not be authorized because the '{subjectEndpointWithValue.Name}' of the resource is empty.");
                     }
 
                     // Perform defensive checks against the remote possibility of SQL injection attack
-                    ValidateTableNameParts(claimEndpointName, targetEndpointName, authorizationSegment.AuthorizationPathModifier);
+                    ValidateTableNameParts(claimEndpointName, subjectEndpointName, authorizationSegment.AuthorizationPathModifier);
 
-                    string derivedAuthorizationViewName = BuildAuthorizationViewName(targetEndpointName, claimEndpointName, authorizationSegment.AuthorizationPathModifier);
+                    string derivedAuthorizationViewName = "auth." + ViewNameHelper.GetAuthorizationViewName(subjectEndpointName, claimEndpointName, authorizationSegment.AuthorizationPathModifier);
```

Comment:
I think it may be worth adding a helper function that includes the schema name. Thoughts about adding ViewHelper.GetAuthorizationViewFullName(...)

Reply:
Done. Added second method, `GetFullyQualifiedAuthorizationViewName` that also uses `SystemConventions.AuthSchema` in place of the string.